data_fixtures/migrations/0008_topics_update_engineering_add_manufacturing.py

```diff
@@ -0,0 +1,59 @@
+"""
+Update the topics; add Manufacturing under Engineering
+This is intended to fix https://github.com/mitodl/hq/issues/5076 - no additional
+mappings since the topics for Principals of Manufacturing include "Manufacturing"
+so it should match on name directly.
+"""
+
+from django.db import migrations
+
+from channels.constants import ChannelType
+from learning_resources.utils import upsert_topic_data_string
+
+map_changes = """
+---
+topics:
+    - icon:
+      id: 4176e385-92c5-4e02-b3cc-4f34d9a4bf40
+      mappings: []
+      name: Manufacturing
+      children: []
+      parent: 952604ab-ae23-45b3-a040-0e5f26fe42df
+"""
+
+
+def add_new_mapping(apps, schema_editor):
+    """Upsert the map_changes data above."""
+
+    upsert_topic_data_string(map_changes)
+
+
+def rollback_new_mapping(apps, schema_editor):
+    """Remove the Manufacturing topic."""
+
+    LearningResourceTopic = apps.get_model(
+        "learning_resources", "LearningResourceTopic"
+    )
+    Channel = apps.get_model("channels", "Channel")
+
+    topic = LearningResourceTopic.objects.filter(
+        parent__topic_uuid="952604ab-ae23-45b3-a040-0e5f26fe42df", name="Manufacturing"
```

Comment:
Not a big deal, but could this match on the topic uuid instead?

```python
topic = LearningResourceTopic.objects.filter(topic_uuid="4176e385-92c5-4e02-b3cc-4f34d9a4bf40").get()
```

Reply:
Fixed this in https://github.com/mitodl/mit-open/pull/1364/commits/17b20709226c96be0cb8da4038467d3aca5ca2b5